helmfile/helmfile.yaml

```diff
@@ -15,12 +15,12 @@ environments:
 releases:
 # Single Tier Website
   - name: {{ requiredEnv "APP_NAME" }}-{{ .Environment.Name }}
-    chart: ../helm/Charts/single-tier-appliation
+    chart: ../helm/Charts/next-template
     values:
-      - overrides/single-tier-application/single-tier-application.yaml.gotmpl
+      - overrides/next-template/next-template.yaml.gotmpl
```

Comment:
The file looks to be renamed to `.../next-template.gotmpl` without the `.yaml`

Reply:
that's weird :/  nice catch though, jeezus

Reply:
I think I confused you.... looks like it should be `.../next-template.yaml.gotmpl`; sorry